server/pipeline/stepbuilder/stepBuilder.go

```diff
@@ -194,6 +195,19 @@ func (b *StepBuilder) genItemForWorkflow(workflow *model.Workflow, axis matrix.A
 		maps.Copy(item.Labels, b.DefaultLabels)
 	}
 
+	// "woodpecker-ci.org" namespace is reserved for internal use
+	for key, _ := range item.Labels {
+		if strings.HasPrefix(key, "woodpecker-ci.org") {
+			delete(item.Labels, key)
+		}
+	}
```

Comment:
Should we at least add a warning to the logs?

Reply:
Added log line for drop pipeline labels